### PR TITLE
⬅️ #360 Fix Back / Next in browser

### PR DESCRIPTION
**Pull request by stephanevieira75**
# Preview:
https://user-images.githubusercontent.com/36481167/111649265-7d65c380-8804-11eb-8d32-02b3038e71f7.mp4

### CODE REVIEW

File: twake/frontend/src/app/services/RouterService.ts [hunk 1]
```diff
@@ -257,6 +257,14 @@ class RouterServices extends Observable {
     }
     return false;
   }
+
+  pushTo(path: string, state?: unknown): void {
```
Comment (chamerling) [inline twake/frontend/src/app/services/RouterService.ts:261]: This method name does not feel natural, why don't we keep `push`?

File: twake/frontend/src/app/services/login/login.js [hunk 2]
```diff
@@ -321,8 +321,8 @@ class Login extends Observable {
         if (!no_reload) {
           Globals.window.location.reload();
         } else {
-          RouterServices.history.push(
-            RouterServices.pathnames.LOGIN + '?' + RouterServices.history.location.search.substr(1),
+          RouterServices.pushTo(
+            `${RouterServices.pathnames.LOGIN}?${RouterServices.history.location.search.substr(1)}`,
```
Comment (chamerling) [inline twake/frontend/src/app/services/login/login.js:325]: We have to `substr` again and again and this looks like a bad API design

File: twake/frontend/src/app/services/RouterService.ts (hunk 1 above)
Comment (stephanevieira75) [reply twake/frontend/src/app/services/RouterService.ts]: Renamed to `push`

File: twake/frontend/src/app/services/login/login.js (hunk 2 above)
Comment (RomaricMourgues) [reply twake/frontend/src/app/services/login/login.js:325]: It is not necessary here, it should be:
```
`${RouterServices.pathnames.LOGIN}${RouterServices.history.location.search}`
```

Comment (stephanevieira75) [reply twake/frontend/src/app/services/login/login.js]: updated 👍🏻

**Pull request merged**